osquery/sql/tests/sqlite_util_tests.cpp

```diff
@@ -215,14 +215,9 @@ TEST_F(SQLiteUtilTests, test_get_query_columns) {
 }
 
 TEST_F(SQLiteUtilTests, test_get_query_tables_failed) {
-  std::string query =
-      "SELECT * FROM time, osquery_info, (SELECT * FROM file) ff GROUP BY pid";
-  std::vector<std::string> tables;
-  auto status = getQueryTables(query, tables);
-  EXPECT_TRUE(status.ok());
-
-  std::vector<std::string> expected = {};
-  EXPECT_EQ(expected, tables);
+  auto dbc = getTestDBC();
+  QueryDataTyped results;
+  EXPECT_FALSE(queryInternal("SELECT * FROM file", results, dbc).ok());
```

Comment:
nit: is there any error message/code we can check to be sure that the failure is caused by a failed constraint?

Reply:
I quickly tested and indeed it seems that from here https://github.com/osquery/osquery/blob/8bab38e30d7ac73738c40860272b2268a07fe72c/osquery/sql/sqlite_util.cpp#L605 we get the "constraint failed" message inside the Status.